tests/testthat/test-multiple_years.R

```diff
@@ -34,7 +34,7 @@ test_that("read multiple years works for episode file", {
   ep <- read_slf_episode(c("1718", "1819"),
     col_select = c("year", "anon_chi")
   ) %>%
-    dplyr::slice_sample(n = 50)
+    dplyr::slice_sample(n = 100)
 
```

Comment:
test-var_lists_match 

This keeps failing to to a mixture of variables not matching, again i think due to the differences in the older files compared to the new update